src/index.js

```diff
@@ -0,0 +1,142 @@
+const state = {
+  tempCounter: 72,
+  lat: '49.7818607',
+  lon: '20.9440505',
+  city: 'Kąśna Dolna, Poland'
+};
+
+const increaseTemp = (event) => {
+  state.tempCounter += 1;
+  const tempCounterContainer = document.getElementById('tempNumber');
+  tempCounterContainer.textContent = state.tempCounter + '°';
+  let tempCircle = document.getElementById('circle');
+  tempCircle = displayColorAndGarden(event);
+};
+
+const decreaseTemp = (event) => {
+  state.tempCounter -= 1;
+  const tempCounterContainer = document.querySelector('#tempNumber');
+  tempCounterContainer.textContent = state.tempCounter +'°';
+  let tempCircle = document.getElementById('circle');
+  tempCircle = displayColorAndGarden(event);
```

Comment:
Notice that lines 18-21 are identical to lines 10-13. 

You can put these 4 lines into a helper method like updateTempAndDisplay() and then call that helper method in the increase/decreaseTemp methods.